### PR TITLE
Improve callout messaging

### LINKED ISSUE #186: Improve messaging for Callout action

### PR DESCRIPTION
**Pull request by mmeldo**
Closes #186

### CODE REVIEW

File: server/game/GameActions/CallOut.js [hunk 1]
```diff
@@ -14,7 +14,7 @@ class CallOut extends GameAction {
 
     createEvent({ caller, callee, isCardEffect = true, canReject = true }) {
         return this.event('onCardCallOut', { caller: caller, callee: callee, isCardEffect: isCardEffect, canReject: canReject }, event => {
-            event.caller.callOut(event.callee, event.canReject);
+            event.caller.callOut(event.callee, event.canReject, event.isCardEffect);
```
Comment (tybar) [inline server/game/GameActions/CallOut.js:17]: Should there be a unit test that handles isCardEffect?

Though looking at this, there's nothing inherent about isCardEffect...is that going to be implemented in the future?

If so, then I think we can wait until that is implemented to write that test.

Comment (mmeldo) [reply server/game/GameActions/CallOut.js:17]: This PR is not yet completed. Usually, when I have PR completed I request a review. We can discuss how we can do it. For example in work, we have a special label that says the PR is ready for review.

Comment (tybar) [reply server/game/GameActions/CallOut.js:17]: That's probably a good way to do that honestly.

**Review by tybar — APPROVED** [review]

**Comment by mmeldo** [comment]
It looks like I will not be able to get to this any time soon. Closing for the time being.

**Pull request closed without merging**